### PR TITLE
Lists page share

### PR DESCRIPTION
**Pull request by laurieboyes**
Pretty basic list sharing overlay functionality. Could do with a design review, but as it's behind the myftLists flag, no harm in releasing.

screen shot 2015-11-24 at 12 48 36

### CODE REVIEW

File: client/components/lists/index.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const Overlay = require('o-overlay');
+
+module.exports = {
+	init: function () {
+
+		const shareButtons = Array.from(document.querySelectorAll('.js-share-list'));
```
Comment (phamann) [inline client/components/lists/index.js]: Is `Array.from` safe?

File: server/lib/lists.js [hunk 2]
```diff
@@ -4,6 +4,10 @@ const myFtApi = require('next-myft-client');
 
 function fetchLists(userId) {
 	return myFtApi.getAllRelationship('user', userId, 'created', 'list')
+		.then(lists => {
+			lists.items.forEach(list => list.publicUrl = 'https://next.ft.com/myft/list/' + list.uuid);
```
Comment (phamann) [inline server/lib/lists.js]: Potentially use a `map` instead?

File: client/components/lists/index.js (hunk 1 above)
Comment (laurieboyes) [inline client/components/lists/index.js]: Works for me locally and looks good on this table: http://kangax.github.io/compat-table/es6/#Array.from

**Comment by phamann** [comment]
:trollface: Would be nice to see `o-share` instead here. But thats for another day :wink:

**Comment by phamann** [comment]
:+1:

**Pull request merged**